languages/csharp/exercises/concept/lists/.docs/introduction.md

```diff
@@ -0,0 +1,7 @@
+Lists in C# are collections of primitive values or instances of structs or classes. They are implemented in the base class library as `List<T>` where `T` is the type of the item in the list. The API exposes a rich set of methods for creating and manipulating lists.
```

Comment:
Maybe also mention that lists can have items added to or removed from them (i.e. they can be resized)? To juxtapose it with arrays.

Reply:
done